skorch/tests/conftest.py

```diff
@@ -78,6 +78,15 @@ def regression_data():
     yt = StandardScaler().fit_transform(y)
     return Xt, yt
 
+@pytest.fixture(scope='module')
+def multi_target_regression_data():
+    X, y = make_regression(
+        1000, 20, n_targets=3, n_informative=10, bias=0, random_state=0)
+    X, y = X.astype(np.float32), y.astype(np.float32).reshape(-1, 1)
```

Comment:
You should not reshape the `y` data to `(-1, 1)`. This will make it a 1-dimensional array, defeating the purpose of multioutput regression.